cmd/thanos/receive.go

```diff
@@ -237,6 +241,19 @@ func runReceive(
 		TSDBStats:         dbs,
 		Limiter:           limiter,
 	})
+	{
```

Comment:
Agree with 1 and 2. 

Regarding 3: yes. So that you can start with an empty config map for the limits configuration, for example, and when you add at least one limiter you don't have to restart your Receive fleet to pick them up.

Reply:
How about a case when I don't specify the flag? Then I would still need to restart the receiver and provide the flag to initiate the limiting. In that case, we don't need to run this job, or am I missing something?

Reply:
I added a check before starting to group that won't start it unless a path is provided. 👍